.github/workflows/ucentralgw-deployment.yaml

```diff
@@ -0,0 +1,125 @@
+name: Update uCentral
+
+defaults:
+  run:
+    shell: bash
+
+env:
+  AWS_EKS_NAME: tip-wlan-main
+  AWS_DEFAULT_OUTPUT: json
+  AWS_DEFAULT_REGION: us-east-2
+  AWS_ACCOUNT_ID: ${{ secrets.AWS_ACCOUNT_ID }}
+  AWS_ACCESS_KEY_ID: ${{ secrets.AWS_CLIENT_ID }}
+  AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_CLIENT_KEY }}
+
+  # DigiCert certs
+  DIGICERT_CERT: ${{ secrets.DIGICERT_CERT }}
+  DIGICERT_KEY: ${{ secrets.DIGICERT_KEY }}
+
+  # https://stackoverflow.com/questions/59977364/github-actions-how-use-strategy-matrix-with-script
+  testbeds: '[
+    {
+      "namespace": "qa",
```

Comment:
@dunaev-opsfleet I think we want to control both the qa and the community "instances"?

Reply:
I've created one environment for now so that Jaspreet and team could test this scenario, so I've only created one. If they approve this setup, I'll add an additional environment and will remove old (numbered) environments